utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -90,6 +91,9 @@ public void cleanTodoTable() {
 		SampleDbConnectionManager.getInstance().closeConnection();
 	}
 
-
-
+	@Given("^clean application state")
+	public void resetState() {
+		int responseCode = TestSupport.getInstance().resetDbWithResponse();
+		Assertions.assertThat(responseCode).isEqualTo(204);
```

Comment:
@tplevko I've gone for checking response code with assert. 
But it might be worth to extend it a bit more in the future to add:
- retry few times if 204 is not returned
- fallback to browser reset
- assert the jsondb state directly.

Reply:
ok, sounds cool, will mark it as TODO for me